tests/libpod/query.go

```diff
@@ -56,6 +56,14 @@ func GetPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance, namespace st
 	return pod, nil
 }
 
+func GetTargetPodForMigration(migration *v1.VirtualMachineInstanceMigration) (*k8sv1.Pod, error) {
+	pod, err := lookupPodBySelector(migration.Namespace, migrationLabelSelector(migration), "")
+	if err != nil {
+		return nil, fmt.Errorf("failed to find pod for migration %s (%s)", migration.Name, migration.UID)
+	}
+	return pod, nil
+}
```

Comment:
How about just using something like:
```
pod, err := lookupPodBySelector(migration.Namespace, migrationLabelSelector(migration), "")
```

instead of having a new public function, i don't think we need this boilerplate